vars/main.yml

```diff
@@ -402,7 +405,15 @@ pg_probackup_instance: "pg_probackup_instance_name"
 pg_probackup_dir: "/mnt/backup_dir"
 pg_probackup_threads: "4"
 pg_probackup_add_keys: "--recovery-target=latest --skip-external-dirs --no-validate"
-pg_probackup_patroni_cluster_bootstrap_command: "pg_probackup-{{ pg_probackup_version }} restore -B {{ pg_probackup_dir }} --instance {{ pg_probackup_instance }} -j {{ pg_probackup_threads }} {{ pg_probackup_add_keys }}"
+# ⚠️ Ensure there is a space at the beginning of each part to prevent commands from concatenating.
+pg_probackup_command_parts:
+  - "pg_probackup-{{ pg_probackup_version }}"
+  - " restore -B {{ pg_probackup_dir }}"
+  - " --instance {{ pg_probackup_instance }}"
+  - " -j {{ pg_probackup_threads }}"
+  - " {{ pg_probackup_add_keys }}"
+pg_probackup_command: "{{ pg_probackup_command_parts | join('') }}"
```

Comment:
I suggest naming the variable as `pg_probackup_restore_command`

Reply:
Done